app/src/lib/uuid.ts

```diff
@@ -1,13 +1,5 @@
 import { randomBytes as nodeCryptoGetRandomBytes } from 'crypto'
-
-// Work around synchronously seeding of random buffer in the v1
-// version of uuid by explicitly only requiring v4. As far as I'm
-// aware we cannot use an import statement here without causing webpack
-// to load the v1 version as well.
-//
-// See
-//  https://github.com/kelektiv/node-uuid/issues/189
```

Comment:
Was this comment removed because we're certain it's not an issue anymore?

Reply:
Oh yes, I should have added a comment! this was solved in electron some versions ago (see https://github.com/electron/electron/issues/2073), I remember suffering this same issue in Atom 😅

We could now change the import to `import { v4 as uuid } from 'uuid';`, but the version of `node-uuid` that we're using does not export ES6 modules so that would cause a bundle size regression.

We should upgrade the `node-uuid` version since newer ones export a ESM version of the library